pkg/generator/keytool.go

```diff
@@ -0,0 +1,193 @@
+package generator
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/ForgeRock/secret-agent/pkg/types"
+	"github.com/pkg/errors"
+)
+
+const (
+	// TODO make these configurable with flags
+	dsKeyMgrPath = "/opt/gen/opendj/bin/dskeymgr"
+	keytoolPath  = "/usr/local/openjdk-11/bin/keytool"
+)
+
+var (
+	tempDir          = ""
+	keystoreFilePath = ""
```

Comment:
why using init() for this?

Reply:
valid question. i think i'm doing this because `ioutil.TempDir` creates a tempdir with a random name when the application starts up, and `keystoreFilePath` needs to contain that name. is there a better way i can do this?

Reply:
(i'm going to submit another PR soon and then ditch this PR, but that PR will have the same change.)